rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -945,7 +945,6 @@ def try_insert_mapping(cur):
         new_ignored_assets_ids = cursor.execute('SELECT value FROM multisettings WHERE name="ignored_asset";').fetchall()  # noqa: E501
         for new_ignored_assets_id in expected_new_ignored_assets_ids:
             assert new_ignored_assets_id in new_ignored_assets_ids
-        assert len(new_ignored_assets_ids) >= 135
```

Comment:
Since spam assets don't get added now on the upgrade and require the data repository they are not here yet. Also the line above this one checks that for the already existing ignored assets their ID gets correctly updated